### ISSUE TITLE
console frontend

### ISSUE THREAD

**Issue by whoizit** [iogf/vy#18]
not planned?

**Comment by iogf** [comment]
Hey, what do you have in mind exactly? you mean a ncurses based interface? if so, yeah. i have that in mind using Tcl Ctk lib.

**Comment by whoizit** [comment]
I mean ncurses ofcourse

**Comment by iogf** [comment]
Yeah, that is planned. in a near future vy would run on console :D
